src/main/java/frc/robot/commands/AcquireCommand.java

```diff
@@ -1,31 +1,47 @@
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.AcquisitionConstants;
 import frc.robot.subsystems.Acquisition;
 
 public class AcquireCommand extends Command {
     private static Acquisition acquisition = Acquisition.getInstance();
+    private Timer timer = new Timer();
+    private double timeToAcquire;
 
-    private double speed;
+    public AcquireCommand() {
+        addRequirements(acquisition);
+        this.timeToAcquire = AcquisitionConstants.defaultTimeToAcquire;
```

Comment:
In the non-timed constructor we usually leave the time to acquire blank. Then if the time is provided we use the timer, otherwise we return false from is finished so that the command continues until the operator stops pressing the button